src/confirmationModal/ConfirmationModal.tsx

```diff
@@ -8,20 +8,34 @@ import { ModalProps } from 'modal/Modal'
 import CrossIcon from 'icon/crossIcon'
 import WarningIcon from 'icon/warningIcon'
 import InfoIcon from 'icon/infoIcon'
+import { assertModalSizes } from '_utils/assert'
 
 export enum ConfirmationModalStatus {
   WARNING = 'warning',
   REMINDER = 'reminder',
 }
 
+export enum ConfirmationModalSize {
+  SMALL = 'small',
+  LARGE = 'large',
+}
+
 export interface ConfirmationModalProps extends ModalProps {
   readonly status: ConfirmationModalStatus
   readonly onConfirm?: () => void
   readonly confirmLabel?: string
   readonly confirmIsLoading?: boolean
+  readonly size?: ModalSize
```

Comment:
ConfirmationModalSize?

Reply:
No, cause this props is passed down to <Modal>